README.md

```diff
@@ -115,7 +115,10 @@ ENVIRONMENT
 if this variable is set when *revgeoc* starts, the program displays received HTTP headers
 
 `OPENCAGE_APIKEY`  
-this mandatory variable must be set in *revgeod*'s environment for it to do reverse geo lookups.
+this optional variable must be set in *revgeod*'s environment for it to do reverse geo lookups via OpenCage.
+
+`LOCATIONIQ_APIKEY`  
+this optional variable must be set in *revgeod*'s environment for it to do reverse geo lookups via [LocationIQ](https://locationiq.com/demo#reverse).
```

Comment:
would you please add a sentence to the effect that either OPENCAGE_APIKEY or LOCATIONIQ_APIKEY are required? As it stands they both appear to be optional :)